pkg/apis/zookeeper/v1beta1/zookeepercluster_types.go

```diff
@@ -613,6 +613,18 @@ type ZookeeperConfig struct {
 	// The default value is 60
 	MaxClientCnxns int `json:"maxClientCnxns,omitempty"`
 
+	// ClientPortListenBacklog The socket backlog length for the ZooKeeper server socket.
+	// This controls the number of requests that will be queued server-side to be processed by the ZooKeeper server.
+	// Connections that exceed this length will receive a network timeout (30s) which may cause ZooKeeper session expiry
```

Comment:
Can we create a map for additional options and parse these options while creating configmap string in `makeZkConfigString`